zone.go

```diff
@@ -231,6 +231,31 @@ type PurgeCacheResponse struct {
 	} `json:"result"`
 }
 
+// ZoneSubscriptionDetails represents all the subscription details for the zone.
+type ZoneSubscriptionDetails struct {
```

Comment:
This struct appears to be missing the `component_values` section. Are you seeing differences in the response to what is documented or has this been left out intentionally?